components-pom/pom.xml

```diff
@@ -267,7 +267,7 @@
     <dependency>
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
-      <version>3.0.1</version>
+      <version>3.0.2</version>
```

Comment:
Can we change to SpotBugs as findbugs is not maintained anymore? see https://gleclaire.github.io/findbugs-maven-plugin/
and https://spotbugs.readthedocs.io/en/latest/maven.html

Reply:
I can't find a spotbugs jsr305 artifact. it's still the most currently used one, see:
https://mvnrepository.com/search?q=jsr305
does it need to be maintained? it simply contains some annotations we use, e.g. `@NotNull`